oap-server/server-library/library-server/src/main/java/org/apache/skywalking/oap/server/library/server/jetty/JettyServer.java

```diff
@@ -133,4 +139,12 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(jettyServerConfig.getHost(), jettyServerConfig.getPort());
     }
+
+    public void  stop() throws ServerException {
+        try {
+            server.stop();
+        } catch (Exception e) {
+            throw new JettyServerException(e.getMessage(), e);
+        }
+    }
```

Comment:
This method is only for testing, we don't usually add methods only for testing, try to use `((org.eclipse.jetty.server.Server)Whitebox.getInternalState(SERVER, "server")).stop();` to do the same thing.

Reply:
you are right, done